wp/lib/bap_wp/src/constraint.mli

```diff
@@ -128,3 +128,10 @@ val eval_model_exn : Z3.Model.model -> z3_expr -> z3_expr
     if retrieving the model fails in the case that [check] was not yet called
     to the solver or the result of the last [check] did not result in SAT. *)
 val get_model_exn : Z3.Solver.solver -> Z3.Model.model
+
+
+type stats
+
+val get_stats : t -> stats -> unit
+
+val print_stats : t -> unit
```

Comment:
Can we add a few comments about `stats`, `get_stats`, and `print_stats` here? What kind of stats is the user expecting?

Reply:
I'm adding these now.